### PR TITLE
chore: add translations for 1.11.0

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This updates the translations needed for our 1.10.0 release.  There are two ways to update translations.
 - You can add suggestions for each translated string. [click here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) for more information on adding suggestions.
 - You can update translations using one of the csvs below by editing the file and re-attaching it to this pull request
    - **de**: [de-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213723/de-locale-update-1.11.0.csv)
    - **es**: [es-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213724/es-locale-update-1.11.0.csv)
    - **fr**: [fr-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213725/fr-locale-update-1.11.0.csv)
    - **it**: [it-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213726/it-locale-update-1.11.0.csv)
    - **ja**: [ja-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213727/ja-locale-update-1.11.0.csv)
    - **ko**: [ko-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213728/ko-locale-update-1.11.0.csv)
    - **pt-br**: [pt-br-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213729/pt-br-locale-update-1.11.0.csv)
    - **zh-cn**: [zh-cn-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213730/zh-cn-locale-update-1.11.0.csv)

**Testing**
 - [x] Verify **de** strings are translated
 - [x] Verify **es** strings are translated
 - [x] Verify **fr** strings are translated
 - [x] Verify **ja** strings are translated
 - [x] Verify **ko** strings are translated
 - [x] Verify **zh-cn** strings are translated
 - [x] Verify **it** strings are translated
 - [x] Verify **pt-br** strings are translated
 - [x] Verify all strings are captured by running the translation check (see https://github.com/sassoftware/vscode-sas-extension/actions/runs/11236482664)

### CODE REVIEW

**Comment by Criptic** [comment]
[de-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17213823/de-locale-update-1.11.0.csv)
German Translation

**Comment by SASJedi** [comment]
[pt-br-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17214471/pt-br-locale-update-1.11.0.csv)

**Comment by vrejany** [comment]
[fr-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17256355/fr-locale-update-1.11.0.csv)
French Translation

File: l10n/bundle.l10n.zh-cn.json [hunk 1]
```diff
@@ -79,6 +79,7 @@
   "SAS 9 Server": "SAS 9 服务器",
   "SAS Compute Context": "SAS计算上下文",
   "SAS Log": "SAS 日志",
+  "SAS Log: {name}": "SAS Log: {name}",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:82]:
```suggestion
  "SAS Log: {name}": "SAS 日志: {name}",
```

**Comment by meilanji** [comment]
[ko-locale-update-1.11.0.csv](https://github.com/user-attachments/files/17291219/ko-locale-update-1.11.0.csv)
Korean translation

File: l10n/bundle.l10n.zh-cn.json (hunk 1 above)
Comment (scottdover) [reply l10n/bundle.l10n.zh-cn.json:82]: Thanks, @scnwwu

**Review by scnwwu — APPROVED** [review]

**Pull request merged**